apps/golden-sample-app/src/app/transactions/transaction-additional-details.component.html

```diff
@@ -0,0 +1,4 @@
+<span class="transaction-account-number">{{ context?.counterPartyAccountNumber }}</span>
+<span class="transaction-my-addition" *ngIf="context?.additions?.['myAddition']">
```

Comment:
something like `context?.additions?.['myAddition']` looks a bit messy in my opinion, any suggestions for cleaning this up a bit?